.github/workflows/build.yml

```diff
@@ -41,13 +41,15 @@ jobs:
       if: "github.ref != 'refs/heads/master'"
 
   pretty:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
     - name: Bootstrap
       run: |
         sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
-        sudo apt-get --no-install-recommends install -y clang-format-10 shellcheck
+        sudo apt-get remove libllvm10
```

Comment:
May I ask why to remove `libllvm10` ?

Reply:
This is a leftover from ubuntu-18.04, where libllvm10 was incomplete and needed to be reinstalled.